internal/initwd/module_install_test.go

```diff
@@ -111,6 +111,67 @@ func TestModuleInstaller_error(t *testing.T) {
 	}
 }
 
+func TestModuleInstaller_symlink(t *testing.T) {
```

Comment:
Is there a test in this func covering a bad symlink? If not, perhaps let's?

Reply:
It's not explicitly covered here, but there are tests that handle other error situations already so I considered that to be sufficient. I could certainly add another test.

Reply:
It probably is sufficient, but I wouldn't mind a pointer so I can go read those to see how they look in practice :)

Reply:
Sure, the test right before this one checks for an expected error:
https://github.com/hashicorp/terraform/blob/35a01283d66451fd3f4ce93957acb33f95a1d41c/internal/initwd/module_install_test.go#L96-L112